plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2021 Twain Byrnes <binaryNewts@google.com>
+ *                    George Popoola <gpopoola@google.com>
+ *
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#include <stdio.h>
+
+#include <unistd.h>
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+        fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+}
+
+gboolean
+fu_plugin_startup (FuPlugin *plugin, GError **error)
+{
+        return TRUE;
+}
+
+static void
+fu_plugin_inhibit_suspend ()
```

Comment:
if you switch to `g_file_set_contents`, this should change return type to `gboolean` and you pass `error`.